src/Hazelcast.Net/DistributedObjects/Impl/HMap.Setting.cs

```diff
@@ -26,11 +26,11 @@ internal partial class HMap<TKey, TValue> // Setting
     {
         /// <inheritdoc />
         public Task SetAsync(TKey key, TValue value)
-            => SetAsync(key, value, TimeToLive.InfiniteTimeSpan, IdleTime.Infinite);
+            => SetAsync(key, value, TimeToLive.Infinite, IdleTime.Infinite);
 
         /// <inheritdoc />
         public Task<TValue> PutAsync(TKey key, TValue value)
-            => PutAsync(key, value, TimeToLive.InfiniteTimeSpan, IdleTime.Infinite);
+            => PutAsync(key, value, TimeToLive.Infinite, IdleTime.Infinite);
```

Comment:
Should be `PutAsync(key, value, TimeToLive.Default, IdleTime.Default);`

Reply:
fixed